generate_engagement_analysis_files.py

```diff
@@ -103,3 +103,54 @@
 
         for row in opted_in_uids_per_show.items():
             writer.writerow(row)
+
+    log.info(f'Computing new and repeat participation for TV episodes ...')
+    # Computes the number of new and repeat consented individuals who participated in the TV_radio shows vis-à-vis
+    # prior radio shows only.
+    # Repeat participants are consented individuals who participated in previous shows prior to the target show.
+    # New participants are consented individuals who participated in target show but din't participate in previous shows.
+    # of rqa_raw_field to participation metrics.
+    tv_radio_shows = ["rqa_s02mag23_raw", "rqa_s02mag24_raw", "rqa_s02mag25_raw",
+                      "rqa_s02mag26_raw", "rqa_s02mag27_raw", "rqa_s02mag28_raw"]
+    tv_radio_shows_uids = set()  # uids of individuals who participated in tv_radio shows.
```

Comment:
```suggestion
    tv_radio_show_uids = set()  # uids of individuals who participated in tv_radio shows.
```

Reply:
mmh since this were 6 shows  `tv_radio_shows_uids` is more appropriate what do you think?

Reply:
Hmm. tv_radio_show_uids instinctively felt like better English, where only the last word would be plural e.g. "the tv and radio show uids" vs. "the tv and radio shows uids". I don't know if it matters that this came from 6 radio shows, I think it's obvious reading the code that this refers to multiple shows.

But I do see your argument, and it's not a big deal either way.

Reply:
If you could get a quick second opinion that might be the fastest way out of this.

Reply:
I think your right tv_radio_show_uids sounds more grammatically correct.